gsl_programmation/templates/includes/_status_form.html

```diff
@@ -1,13 +1,13 @@
-<form action="{% url 'programmation:patch-simulation-projet-status' %}"
-      method="POST">
+{% load gsl_filters %}
+<form hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}'
+      hx-swap="outerHTML"
+      hx-patch="{% url 'programmation:patch-simulation-projet-status' simu.id %}"
+      hx-trigger="change"
+      hx-disabled-elt="find #status-select">
```

Comment:
Attention de garder les attributs "normaux" pour que ça continue de marcher sans HTMX
```suggestion
<form hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}'
      hx-swap="outerHTML"
      hx-patch="{% url 'programmation:patch-simulation-projet-status' simu.id %}"
      hx-trigger="change"
      hx-disabled-elt="find #status-select"
      action="{% url 'programmation:patch-simulation-projet-status' %}"
      method="POST">
```

Reply:
Ah oui, l'idée serait de faire en sorte que quand tu enlèves le JS de la page, alors ça fonctionne toujours ?

Reply:
Yes, amélioration progressive :)

Reply:
Après, j'ai viré le déclenchement onchange, donc je crois que ça ne fonctionne plus sans JS.
Je me demande si on peut garder le fonctionnement sans JS 🤔

Reply:
En fait, c'est important ce sujet. Parce qu'on peut virer toute la partie "form" si on se base que sur HTMX.
Donc si on se base que sur HTMX, ça peut aussi alléger le code !

Reply:
La manière dont je recommande l'adoption des technologies comme HTMX, c'est vraiment l'amélioration progressive. Côté HTMX, ça implique souvent d'utiliser `hx-boost` plutôt que des attributs ad-hoc. Enlever un `<form>` a aussi un impact sur l'accessibilité (les formulaires ont plein de comportement par défaut comme la soumission quand on presse Entrée pour n'en citer qu'un).

https://htmx.org/docs/#progressive_enhancement

Reply:
Ok, ça roule !
C'est bon, j'ai trouvé comment faire marcher le formulaire sans JS sans que ça vienne embêter le JS.
Merci à `<input type="submit" hidden />` (cf [so](https://stackoverflow.com/questions/477691/submitting-a-form-by-pressing-enter-without-a-submit-button))
Merci !